strapi/strapi
# [Data transfer] Add telemetry to transfer command

### What does it do?

Adds telemetry events to the `transfer` command

Refactors the `transfer` code to more closely match the format of ìmport` and `export`

### Why is it needed?

Telemetry events are missing from transfer

### How to test it?

Run a `strapi transfer` command and telemetry events should be sent on start, finish, and error.

### Related issue(s)/PR(s)
